### ISSUE TITLE
Dataset summary files not correctly reporting number of records

### ISSUE THREAD

**Issue by Mesibov** [gbif/portal-feedback#3980]
From [this page](https://www.gbif.org/dataset/search?q=) GBIF users can download summary files of datasets: either all datasets, or occurrence, checklist or sampling event separately.

Both the "all" and "occurrence" dataset summary files incorrectly report 0 records for thousands of datasets. Today the "all" download lists 2897 occurrence datasets with zero records. All of the ones I've checked in fact have records. To take one of 2897 examples, the Macquarie Island Elephant Seal Sightings occurrence record dataset, dataset_key 82e218a4-f762-11e1-a439-00145eb45e9a, is listed in the summary file as having zero records, but the dataset's [webpage](https://www.gbif.org/dataset/82e218a4-f762-11e1-a439-00145eb45e9a) says there are 30998 occurrences, with a publication date of 22 March 2019 and a metadata update on 15 February 2022.

Is this an update issue in the routine that generates the summary files? How else can GBIF data users get current numbers of records in blocks of datasets, without checking datasets individually?

**Comment by MortenHofft** [comment]
This sounds like the same as https://github.com/gbif/portal-feedback/issues/3458

**Comment by Mesibov** [comment]
@MortenHofft  Agree, it looks the same. How else can GBIF data users get current numbers of records in blocks of datasets, without checking datasets individually?
